### PR TITLE
=doc Fix wrong scala/java snippet directives #23513

### PR DESCRIPTION
**Pull request by jlprat**
Issue: #23513
Fix inline snippet in stream-io
Fix wrong rendering on other pages

### CODE REVIEW

File: akka-docs/src/main/paradox/scala/logging.md [hunk 1]
```diff
@@ -254,7 +254,9 @@ loggers, which are based upon the originating object’s class name as log
 category. The override of `getClazz` is only included for demonstration
 purposes as it contains exactly the default behavior.
 
-@@@ note
+@@@
```
Comment (jlprat) [inline akka-docs/src/main/paradox/scala/logging.md]: This is the end of the `@@@ div { .group-scala }`

File: akka-docs/src/main/paradox/scala/stream/stream-flows-and-basics.md [hunk 2]
```diff
@@ -254,7 +254,8 @@ but is not restricted to that—it could also mean opening files or socket conne
 
 Materialization is triggered at so called "terminal operations". Most notably this includes the various forms of the `run()`
 and `runWith()` methods defined on `Source` and `Flow` elements as well as a small number of special syntactic sugars for running with
-well-known sinks, such as @scala[`runForeach(el => ...)`] @java[`runForeach(el -> ...)`] (being an alias to @scala[`runWith(Sink.foreach(el => ...))`] @java[`runWith(Sink.foreach(el -> ...))`].
+well-known sinks, such as @scala[`runForeach(el => ...)`]@java[`runForeach(el -> ...)`]
+(being an alias to @scala[`runWith(Sink.foreach(el => ...))`]@java[`runWith(Sink.foreach(el -> ...))`]).
```
Comment (jlprat) [inline akka-docs/src/main/paradox/scala/stream/stream-flows-and-basics.md]: Having a parenthesis after the `@java[]` directive caused to render it verbatim

File: akka-docs/src/main/paradox/scala/stream/stream-io.md [hunk 3]
```diff
@@ -23,7 +23,7 @@ Java
 Next, we simply handle *each* incoming connection using a `Flow` which will be used as the processing stage
 to handle and emit `ByteString` s from and to the TCP Socket. Since one `ByteString` does not have to necessarily
 correspond to exactly one line of text (the client might be sending the line in chunks) we use the @scala[`Framing.delimiter`]@java[`delimiter`]
-helper Flow @scala[]@java[from `akka.stream.javadsl.Framing`] to chunk the inputs up into actual lines of text. The last boolean
+helper Flow @java[from `akka.stream.javadsl.Framing`] to chunk the inputs up into actual lines of text. The last boolean
```
Comment (jlprat) [inline akka-docs/src/main/paradox/scala/stream/stream-io.md]: Empty `@scala[]` directives are rendered verbatim

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**